lib/traject/macros/iiif.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+module Macros
+  # Macros for extracting Stanford Specific MODS values from Nokogiri documents
+  module Stanford
+    def grab_sul_iiif_links(manifest)
+      manifest = Net::HTTP.get(URI.parse(manifest))
+      JSON.parse(manifest)
+    rescue
+      nil
```

Comment:
Because this could return nil, then we should test that the other methods that take this result as a parameter are able to work when you pass them `nil`, right?